pyproject.toml

```diff
@@ -53,12 +52,13 @@ sphinx = {version="*", optional=true}
 sphinxcontrib-openapi = {version="^0.8.3", optional=true}
 mypy = {version="1.10.1", optional=true}
 requests = {version = "^2.32.3", optional = true}
+scipy = "^1.15.2"
```

Comment:
If this is required can you move it up into the section above this.

Also, this pins us to 1.15 versions only. Can we change this to `^1.13` to get a few allowable versions. Or at least `^1.15` to avoid the `.2` portion limiting our updates in only the patch version specifier.